tests/unit/test_download.py

```diff
@@ -127,7 +127,12 @@ def test_path_to_url_unix():
 
 @pytest.mark.skipif("sys.platform == 'win32'")
 def test_url_to_path_unix():
```

Comment:
I think this test should be moved to precede `test_url_to_path_win()` so that you can more easily compare the Windows test cases with the Unix ones.